frontend/util/fetching.ts

```diff
@@ -76,3 +86,15 @@ export async function backendPost<T extends keyof postRequests>(
 		postRequests[T]["response"]
 	>;
 }
+
+export async function backendFormPost<T extends formPostRequests>(
```

Comment:
With the new changes in fetching, I can see that since the request type for resume-upload has been changed to FormData (to handle and upload the contents of the file). And of course, since this would change the request type in resume-upload, the backendFormPost would allow the FormData to be accepted.